src/app/config/config.ts

```diff
@@ -206,6 +213,7 @@ const config: Config = {
   db: dbConfig,
   aws: awsConfig,
   mail: mailConfig,
+  sgid,
```

Comment:
if we move the `sgid` instantiation to its own file, then we will not need to add it into the exported `config` constant and `sgid.service` can import directly